tests/test_dht_experts.py

```diff
@@ -0,0 +1,76 @@
+import random
+import uuid
+from itertools import chain
+
+import hivemind
+from hivemind import LOCALHOST
+
+
+def test_hivemind_dht():
+    peers = [hivemind.DHT(start=True)]
+    for i in range(10):
+        neighbors_i = [f'{LOCALHOST}:{node.port}' for node in random.sample(peers, min(3, len(peers)))]
+        peers.append(hivemind.DHT(initial_peers=neighbors_i, start=True))
+
+    you: hivemind.dht.DHT = random.choice(peers)
+    theguyshetoldyounottoworryabout: hivemind.dht.DHT = random.choice(peers)
```

Comment:
Option 1: snake case :)
Option 2: other names for nodes?

Reply:
If we are to change it, i suggest we change it everywhere in a separate PR

Reply:
Let's create an issue for that as well (maybe collect all reformatting concerns in a single one)